docs/docs/langfuse-exporter.md

```diff
@@ -0,0 +1,76 @@
+# Langfuse Exporter
+
+Koog provides built-in support for exporting agent traces to [Langfuse](https://langfuse.com/), a platform for observability and analytics of AI applications.  
+With Langfuse integration, you can visualize, analyze, and debug how your Koog agents interact with LLMs, APIs, and other components.
+
+For background on Koog’s OpenTelemetry support, see the [OpenTelemetry support](https://docs.koog.ai/opentelemetry-support/).
+
+---
+
+### Setup Instructions
+
+1. Create a Langfuse project. Follow the setup guide at [Create new project in Langfuse](https://langfuse.com/docs/get-started#create-new-project-in-langfuse)
+2. Obtain API credentials. Retrieve your Langfuse `public key` and `secret key` as described in [Where are Langfuse API keys?](https://langfuse.com/faq/all/where-are-langfuse-api-keys)
+3. Set environment variables. Add the following variables to your environment:
+
+```bash
+   export LANGFUSE_HOST="https://cloud.langfuse.com"
+   export LANGFUSE_PUBLIC_KEY="<your-public-key>"
+   export LANGFUSE_SECRET_KEY="<your-secret-key>"
+```
+
+Once configured, Koog automatically forwards OpenTelemetry traces to your Langfuse instance.
+
+## Configuration
+
+To enable Langfuse export, install the **OpenTelemetry feature** and add the `LangfuseExporter`.  
+The exporter uses `OtlpHttpSpanExporter` under the hood to send traces to Langfuse’s OpenTelemetry endpoint.
+
+### Example: Agent with Langfuse Tracing
+
+```kotlin
```

Comment:
Could you include knit tags please, so kotlin examples will be compilable (guideline is [here](https://github.com/JetBrains/koog/tree/develop/docs#how-to-annotate-docs))? Or can I do it for you?